integration-cli/docker_cli_pull_local_test.go

```diff
@@ -54,9 +52,7 @@ func (s *DockerRegistrySuite) TestConcurrentPullWholeRepo(c *check.C) {
 		    ENV BAR bar
 		    CMD echo %s
 		`, repo), true)
-		if err != nil {
-			c.Fatal(err)
-		}
+		c.Assert(err, check.IsNil, check.Commentf("%v", err))
```

Comment:
No need for the Commentf here since the error output will include the value of `err` already. But if there is something being output by this command, then that'd be really awesome here.